File: app/src/main/java/com/lofts/dictionary/bean/Result.java
package com.lofts.dictionary.bean;

import java.util.List;

public class Result {

    private String word;
    private Pronunciation pronunciation;
    private List<Define> defs;
    private List<Sample> sams;

    public Result() {
    }

    public String getWord() {
        return word;
    }

    public void setWord(String word) {
        this.word = word;
    }

    public Pronunciation getPronunciation() {
        return pronunciation;
    }

    public void setPronunciation(Pronunciation pronunciation) {
        this.pronunciation = pronunciation;
    }

    public List<Define> getDefs() {
        return defs;
    }

    public void setDefs(List<Define> defs) {
        this.defs = defs;
    }

    public List<Sample> getSams() {
        return sams;
    }

    public void setSams(List<Sample> sams) {
        this.sams = sams;
    }
}

File: app/src/main/java/com/lofts/dictionary/bean/Result.kt
package com.lofts.dictionary.bean

/**
 * PACKAGE_NAME：com.lofts.dictionary.bean
 * DATE：2018-12-18 23:08
 * USER: asus
 * DESCRIBE:
 */
data class Result(
    var word: String,
    var pronunciation: Pronunciation,
    var defs: List<Define>,
    var sams: List<Sample>
)